From 47c6af021e3eb55cfa502e1387ac75f8e522e234 Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Tue, 14 Feb 2017 12:14:16 +0000
Subject: [PATCH 1/3] mx6sx: udoo_neo: Define the default serial console

Standard boot processes including distro boot generally expect the
default console to be defined.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---
 include/configs/udoo_neo.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/udoo_neo.h b/include/configs/udoo_neo.h
index 23a3685..5b46272e 100644
--- a/include/configs/udoo_neo.h
+++ b/include/configs/udoo_neo.h
@@ -30,6 +30,7 @@
 /* Linux only */
 #define CONFIG_ENV_VARS_UBOOT_RUNTIME_CONFIG
 #define CONFIG_EXTRA_ENV_SETTINGS \
+	"console=ttymxc0,115200\0" \
 	"fdt_high=0xffffffff\0" \
 	"initrd_high=0xffffffff\0" \
 	"fdtfile=undefined\0" \
-- 
2.9.3

